app/tests/components/Form/ApplicationForm.test.tsx

```diff
@@ -68,6 +68,11 @@ const submissionPayload = {
           submissionCompletedBy: 'test',
           submissionTitle: 'test',
         },
+        acknowledgements: {
+          acknowledgementsList: [
+            1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17,
+          ],
```

Comment:
`acknowledgementsList` is an array of strings. You could refactor this to use `acknowledgementsEnum` once #709 is merged